src/Controllers/NIPSWeb/default.php

```diff
@@ -30,7 +31,9 @@
 }
 
 CoreUtils::getTemplateObject()->setTemplate($template)
-    ->addVariable('title', $title)
+    ->addVariable('title', "Show Planner")
+    ->addVariable('show_title', $show_title)
```

Comment:
timeslotname is currently globally exposed - is it worth doing so with these and removing timeslotname, instead of having kind-of-duplication?